src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy

```diff
@@ -85,30 +92,28 @@ class SwitchSyncSpec extends HealthCheckSpecification {
         def switchPair = switchPairs.all().nonNeighbouring().random()
 
         and: "Create an intermediate-switch flow"
-        def flow = flowHelperV2.randomFlow(switchPair)
-        flowHelperV2.addFlow(flow)
+        def flow = flowFactory.getRandom(switchPair)
 
         and: "Drop all rules an meters from related switches (both default and non-default)"
-        def involvedSwitches = pathHelper.getInvolvedSwitches(flow.flowId)
-        def involvedSwitchIds = involvedSwitches*.getDpId()
-        def cookiesMap = involvedSwitches.collectEntries { sw ->
-            [sw.dpId, northbound.getSwitchRules(sw.dpId).flowEntries.findAll {
-                !(it.cookie in sw.defaultCookies)
+        def involvedSwitches = flow.retrieveAllEntityPaths().getInvolvedSwitches()
```

Comment:
let's use the same approach to get List<Switch> involvedSwitches as pathHelper has :
```
List<Switch> involvedSwitches = flow.retrieveAllEntityPaths().flowPath.getInvolvedIsls()
        .collect { [it.srcSwitch, it.dstSwitch] }.flatten().unique() as List<Switch>
 ```
Could you please delete new overload methods in SwitchHelper with `switchId` as param?

Reply:
Done + removed the otherloaded methods